src/utils/utils.go

```diff
@@ -0,0 +1,34 @@
+package utils
+
+import "strings"
+
+func ParseArgs(args []string) map[string][]string {
```

Comment:
I don't think we'll ever call this form outside subcommand.go so we should leave it there IMO. Having a utils file like this is generally considered something best avoided. If there is truly a stateless function that is called from a bunch of different files a utils file would be ok, but it's preferable in cases like these to put ParseArgs in the file where it's used.

Also, once you move it back to subcommand.go, you should make it unexported ```func parseArgs``` since we don't expect other classes to call it.

Reply:
done